make/package-kube

```diff
@@ -9,7 +9,7 @@ ARTIFACT=uaa-kube-${GIT_DESCRIBE}.zip
 
 rm ${GIT_ROOT}/${ARTIFACT} 2>/dev/null || true
 
-cd kube && zip -r9 ${GIT_ROOT}/${ARTIFACT} * \
+cd kube && zip -r9 ${GIT_ROOT}/${ARTIFACT} * ../kube-test/* \
```

Comment:
I was mistaken. The SCF `kube-dist` target runs SCF `kube`, which runs SCF `uaa-kube`, which is UFR `kube`. Nowhere in this is UFR `kube-dist` called, which is responsible for the copying above. The `SCF `kube-dist` command has to be extended to either run the UFR `kube-dist` (generating a superfluous zip), or doing the copying itself.

I will now make a fix. For ease of tracking dependencies I will have it run the UFR `kube-dist`.